packages/react/src/authentication/hooks/__tests__/useSocialLogin.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { cleanup, renderHook } from '@testing-library/react';
+import { mockInitialState } from 'tests/__fixtures__/authentication/index.mjs';
+import { mockStore } from '../../../../tests/helpers/index.js';
+import { postAccountLink, postSocialLogin } from '@farfetch/blackout-client';
+import { Provider } from 'react-redux';
+import { useSocialLogin } from '../index.js';
+
+const mockLoginData = {
+  provider: 'Google',
+  socialAccessToken: 'xxx-xxx-xxx-xxx',
+  rememberMe: true,
+  countryCode: 'PT',
+};
+
+const mockAccountLinkData = {
+  username: 'xxxxxxxx',
+  password: 'xxxxxxxx',
+};
+
+jest.mock('@farfetch/blackout-client', () => {
+  const original = jest.requireActual('@farfetch/blackout-client');
+
+  return {
+    ...original,
+    postSocialLogin: jest.fn(() => Promise.resolve()),
+    postAccountLink: jest.fn(() => Promise.resolve()),
+  };
+});
+
+const genericMock = {
+  actions: {
+    accountLink: expect.any(Function),
+    login: expect.any(Function),
+  },
+};
+
+const getRenderedHook = (state = mockInitialState) => {
+  const {
+    result: { current },
+  } = renderHook(() => useSocialLogin(), {
+    wrapper: props => <Provider store={mockStore(state)} {...props} />,
+  });
+
+  return current;
+};
+
+describe('useSocialLogin', () => {
+  beforeEach(jest.clearAllMocks);
+
+  afterEach(cleanup);
+
+  it('should return correctly with initial state and call all hook dependencies with the correct options', () => {
```

Comment:
Can this test description be reviewed, please? 
This hook does not have other hook dependencies (apart from `useAction`) and you're expecting that nothing gets called at this point.